### PR TITLE
Correct beug on parsing twelvehour date

### PR DESCRIPTION
**Pull request by strokyl**
Before this commit, when using clockpicker in twelvehour mode, minute and AM/PM was not parsed correctly.

### CODE REVIEW

File: dist/bootstrap-clockpicker.js [hunk 1]
```diff
@@ -453,19 +453,27 @@
...
+		if (trimedValue === 'now') {
 			var now = new Date(+ new Date() + this.options.fromnow);
-			value = [
-				now.getHours(),
-				now.getMinutes()
-			];
-		}
-		this.hours = + value[0] || 0;
-		this.minutes = + value[1] || 0;
+      this.hours = now.getHours();
```
Comment (Eriflow) [inline dist/bootstrap-clockpicker.js:446]: There is an small indent issue. Mb fix the indent for the whole file in a specific commit ?

File: src/clockpicker.js [hunk 2]
```diff
@@ -453,19 +453,27 @@
 			this.isAppended = true;
 		}
 
-		// Get the time
-		var value = ((this.input.prop('value') || this.options['default'] || '') + '').split(':');
-		if (value[0] === 'now') {
+    var trimedValue = ((this.input.prop('value') || this.options['default'] || '') + '').replace(/^\s\s*/, '').replace(/\s\s*$/, '');
+
+		if (trimedValue === 'now') {
```
Comment (GrahamLinagora) [inline src/clockpicker.js]: weird indent

**Comment by strokyl** [comment]
+1

**Comment by GrahamLinagora** [comment]
:+1:

**Comment by strokyl** [comment]
arf if was his file which wasn't indented correctly.

**Comment by Eriflow** [comment]
:+1:

**Pull request merged**